src/Orckestra.Composer/ViewModels/ViewModelMapper.cs

```diff
@@ -280,12 +280,16 @@ private object LocalizeValue(object value, IPropertyMetadata propertyMetadata, C
             }
             else
             {
-                throw new InvalidOperationException(
-                    string.Format(
-                        "Cannot map property '{0}', because source property type was '{1}' and viewmodel property type was '{2}'.",
-                        viewModelPropertyName,
-                        sourceType.FullName,
-                        viewModelPropertyType.FullName));
+                try
+                {
+                    viewModelPropertyValue = Convert.ChangeType(sourceValue, viewModelPropertyType);
+                }
+                catch
+                {
+                    throw new InvalidOperationException(
```

Comment:
$"Cannot map property '{viewModelPropertyName}', because the source property type is '{sourceType.FullName}' " +
$"and the target ViewModel property type is '{viewModelPropertyType.FullName}'.